jgeorge37/acmwosu-db
# ADA-41 Company search input

Added company search input component in the "Options" form and "Add Contact" form. Utilizes API endpoint /api/company/byString for searching companies. 

To test:

- Get local postgres instance if you don't already have one
- See the readme changes from ADA-40 for the other changes you have to make to work with a local DB
- Add companies to your local db
- npm run dev
- Go to localhost:3000/companycontacts and click on either the "Options" button or "Add Contact". There will be a component to search for a company
- You should get company names and ids multiselect component that correspond with the input you entered, updated every time you enter a character
